pkgs/by-name/au/authentik/package.nix

```diff
@@ -30,7 +30,6 @@ let
     homepage = "https://goauthentik.io/";
     license = licenses.mit;
     platforms = platforms.linux;
-    broken = stdenvNoCC.hostPlatform.isAarch64;
```

Comment:
Could you add `broken = stdenvNoCC.buildPlatform != stdenvNoCC.hostPlatform` while we are here?

It fails to pass along the correct compiler configuration: https://paste.fliegendewurst.eu/PkWbUa.log

Reply:
Done